liberty-maven-plugin/src/it/verify-user-feature-it/src/test/java/net/wasdev/wlp/test/feature/it/VerifyUsrFeatureTest.java

```diff
@@ -46,5 +50,54 @@ public void testVerifyUsrFeature() throws Exception {
             throw new AssertionError ("Fail to install user feature.", e);
         }
     }
+    
+    
+
+    @Test
+    public void buildLogCheck() throws Exception {
+        File buildLog = new File("build.log");
+        assertTrue(buildLog.exists());
+
+        InputStream buildOutput = null;
+        InputStreamReader in = null;
+        Scanner s = null;
+
+        final String VERIFIED_MESSAGE = "All features were successfully verified.";
+        boolean VERIFIED_MESSAGE_FOUND = false;
+
+        final String CWWKF1514E_MESSAGE = "CWWKF1514E";
+        boolean CWWKF1514E_MESSAGE_FOUND = false;
+        
+        final String CWWKF1508E_MESSAGE = "CWWKF1508E";
+        boolean CWWKF1508E_MESSAGE_FOUND = false;
+        
+        final String CWWKF1512E_MESSAGE = "CWWKF1512E";
+        boolean CWWKF1512E_MESSAGE_FOUND = false;
+        
+        
+
+        try {
```

Comment:
Should use a try-with-resources so that the stream is closed automatically. I realize you may have copied this, which means I need to go fix other test cases.